installing_on_kubernetes/_topics/prepare_kubernetes_namespace.md

```diff
@@ -27,7 +27,11 @@ The operator directory referenced below can be found [here]({{ site.data.product
    $ oc create -f config/rbac/service_account.yaml
    ```
 
-4. Deploy the operator in your namespace.
+4. Install Strimzi Kafka
+
+{% include_relative install_kafka.md %}
```

Comment:
This is a little weird, but I'm not sure how to do it otherwise.  The reason it's weird is because the header title will appear under the number, and might break the numbered bulleting.  We'd have to see what a rendered page looks like.

Additionally, I'm not sure if we use this downstream, but if so, it needs to be productized.